### PR TITLE
Update botttom_nav_cubit.dart and main.dart

### PR DESCRIPTION
**Pull request by gaganyadav80**
Added a function in BottomNavCubit to make the code shorter and cleaner.
removed the deprecated title widget from BottomNavigationBarItem and added label String.
Commented on the code which can also make the _buildBody() shorter and cleaner.

This tutorial helped me understand the concept so thought of contributing if it helps. 
Thank you for the simple and clean solution of BottomNavBar with cubit, it was exactly what I was looking for.

### CODE REVIEW

**Review by wisnuwiry — COMMENTED** [review]
Thanks for open PR @gaganyadav80 , Please remove unused code (Not documentation code). Then when done I can merge this PR

**Comment by MashudiSudonym** [comment]
thank you, you are awesome.

but, you can remove condition for check range of array.

File: lib/main.dart [hunk 1]
```diff
@@ -53,42 +53,22 @@ class _MainPageState extends State<MainPage> {
...
-        return NotificationPage();
-        break;
-      case 4:
-        return ProfilePage();
-        break;
-      default:
-        return Text('Not Found');
-    }
+    if (index <= 4)
+      return _pageNavigation[index];
```
Comment (MashudiSudonym) [inline lib/main.dart:69]: you can just return
`_pageNavigation.elementAt(index)`

without checking the index range.

File: lib/main.dart [hunk 2]
```diff
@@ -107,29 +87,9 @@ class _MainPageState extends State<MainPage> {
   }
 
   void _getChangeBottomNav(int index) {
-    /// Only one fuction to update the index
     if (index <= 4)
       context.cubit<BottomNavCubit>().updateIndex(index);
```
Comment (MashudiSudonym) [inline lib/main.dart:91]: this code it's same 
you can remove the condition of check index range, and just add 
`context.cubit<BottomNavCubit>().updateIndex(index);`

**Review by MashudiSudonym — COMMENTED** [review]
hei, check my review

**Pull request merged**